docs/gettinginvolved/authors.rst

```diff
@@ -1 +1,7 @@
-.. include:: ../../AUTHORS.rst
+=======
+Authors
+=======
+
+.. ifconfig:: is_spelling_check is False
```

Comment:
`not is_spelling_check`?  Identity test against bool is not very idiomatic Python (maybe this isn't Python?)